sympy/sets/tests/test_sets.py

```diff
@@ -423,6 +423,8 @@ def test_contains():
     assert Interval(0, 2, True, True).contains(0) is S.false
     assert Interval(0, 2, True, True).contains(2) is S.false
 
+    assert Interval(0, 2) in Interval(0, 2) is S.false
```

Comment:
Your tests are failing.

I think, it should be `(Interval(0, 2) in Interval(0, 2)) == S.false` 

You need to add parenthesis and `==` sign instead of `is`.

```
In [17]: type(Interval(0,2).contains(Interval(0, 2)))
Out[17]: sympy.logic.boolalg.BooleanFalse

In [18]: type(Interval(0,2) in Interval(0, 2))
Out[18]: bool
```

`S.false` and `False` are both different object, so `is` operator will return `False`